benchmarks/TFLite/linux-riscv.cmake

```
@@ -0,0 +1,85 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RV64_GENERIC_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+v"
+  "--riscv-v-vector-bits-min=512"
```

Based on CI RV64 QEMU config.
```suggestion
  "--riscv-v-vector-bits-min=256"
```

---

Does the same thing apply to RV32?

---

For RV32, it is okay to keep the value as 512

---

We are targeting RV64 with 512 bits so I think we should keep 512. We don't plan to use QEMU for now but would it be possible to update the configuration to 512?